src/providers/be_refresh.c

```diff
@@ -35,7 +35,8 @@ static errno_t be_refresh_get_values_ex(TALLOC_CTX *mem_ctx,
                                         struct ldb_dn *base_dn,
                                         const char *key_attr,
                                         const char *value_attr,
-                                        char ***_values)
+                                        char ***_values,
+                                        int optflags)
```

Comment:
Nit: I would prefer if the parameters that are written to (output parameters, those that start with an underscore) could stay as the last ones in the list.

Reply:
True, I will fix this.

Reply:
It looks like this wasn't fixed in the last version

Reply:
Fixed now